components/observations/uploader/ObservationsUploader.test.ts

```diff
@@ -11,6 +11,14 @@ import {uploadImage as uploadImageOriginal} from 'components/observations/upload
 import {logger} from 'logger';
 import {AvalancheCenterID, MediaItem, MediaType, MediaUsage} from 'types/nationalAvalancheCenter';
 
+jest.mock('react-native/Libraries/LogBox/LogBox', () => ({
```

Comment:
Unclear why we need this or what we can do to avoid this mess, but without the mock, locally my `jest` run fails with

```
 FAIL  components/observations/uploader/ObservationsUploader.test.ts
  ● Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'ignoreAllLogs')

      19 |
      20 | if (process.env.EXPO_PUBLIC_DISABLE_LOGBOX) {
    > 21 |   LogBox.ignoreAllLogs(true);
         |          ^
      22 | }
      23 |
      24 | const defaultLogLevel = process.env.NODE_ENV === 'test' ? 'WARN' : 'INFO';

      at Object.<anonymous> (logger.ts:21:10)
      at Object.<anonymous> (components/observations/uploader/uploadObservation.ts:5:1)
      at Object.<anonymous> (components/observations/uploader/ObservationsUploader.ts:19:1)
      at Object.<anonymous> (components/observations/uploader/ObservationsUploader.test.ts:8:1)
```